packages/vue/src/index.js

```diff
@@ -0,0 +1,79 @@
+const web = require('@neutrinojs/web');
+const path = require('path');
+const arrify = require('arrify');
+const merge = require('deepmerge');
+
+const LOADER_EXTENSIONS = /\.vue$/;
+const MODULES = path.join(__dirname, 'node_modules');
+
+module.exports = (neutrino, options) => {
+  const lintRule = neutrino.config.module.rules.get('lint');
+  const compileRule = neutrino.config.module.rules.get('compile');
+
+  neutrino.config.module.rule('vue')
+    .test(LOADER_EXTENSIONS)
+    .use('vue')
+    .loader(require.resolve('vue-loader'))
+    .options(options);
+
+  if (compileRule && compileRule.uses.has('babel')) {
```

Comment:
I'm confused about what this does. These isn't a compile loader here until `neutrino.use(web, options)`.